modules/javafx.graphics/src/main/java/javafx/print/Printer.java

```diff
@@ -86,11 +86,13 @@ private static ReadOnlyObjectWrapper<Printer> defaultPrinterImpl() {
         if (security != null) {
             security.checkPrintJobAccess();
         }
+        Printer p = PrintPipeline.getPrintPipeline().getDefaultPrinter();
         if (defaultPrinter == null) {
-            Printer p = PrintPipeline.getPrintPipeline().getDefaultPrinter();
             defaultPrinter =
                 new ReadOnlyObjectWrapper<Printer>(null, "defaultPrinter", p);
-        }
+        } else {
+        defaultPrinter.setValue(p);
+    }
```

Comment:
Minor: indentation is off for these two lines

Reply:
That's what you get for trying to use an IDE ... will fix ..